### ISSUE TITLE
Meteor 1.6.1 Injected data not available in browsers

### ISSUE THREAD

**Issue by mhagmajer** [abecks/meteor-inject-data#5]
https://github.com/meteor/meteor/issues/9708

**Comment by abecks** [comment]
It doesn't look like I'm going to be able to fix this. There are so many ways to do SSR now that I can't find a way to make this work for all situations. Since we no longer want to attach data to the `response` object, I tried to use Meteor's `Meteor.EnvironmentVariable` to persist the data, but it only works if I change InjectData to a very specific API with a wrapper function:

```
InjectData.init(function() {
   // you can only use InjectData.pushData within this context, you would essentially have to wrap your entire app in it
})
```

Which feels like a huge departure from how this package used to work. I'm open to suggestions and PRs.

**Comment by abecks** [comment]
Found another way, this should be fixed in 2.2.0
